_posts/2013-04-09-asset-compilation.md

```diff
@@ -21,31 +21,48 @@ sure to check out all the options and usage notes.
 
 ### JS Transpiling
 
-Ember-cli automatically transpiles future javascript (ES6/ES2015, ES2016 and beyond) into standard ES5
-javascript that runs on every browser using [Babel JS](https://babeljs.io) with the [ember-cli-babel](https://github.com/babel/ember-cli-babel) addon.
+Ember CLI automatically transpiles future JavaScript (ES6/ES2015, ES2016 and beyond) into standard ES5
+JavaScript that runs on every browser using [Babel JS](https://babeljs.io) with the [Ember CLI Babel](https://github.com/babel/ember-cli-babel) addon.
 
-The default configuration handles most project's needs, but you can provide options to the transpile
-to disable specific transformations if by example your app only targets ES6 capable browsers, or
-on the contrary enable transpilation of some experimental features that not enabled by default just yet.
+Internally, Ember CLI Babel uses `babel-preset-env`, which figures out which parts of your code 
+need to be transpiled to ES5 by **analyzing your project's browser support targets**. A `target` is a special keyword
+that maps to a [browserlist](https://github.com/ai/browserslist) support rule. These are defined in your
+`config/targets.js` file, which [Ember CLI generates](https://github.com/ember-cli/ember-cli/blob/master/blueprints/app/files/config/targets.js) like so:
 
-You can configure how babel works using the `babel` option in  `ember-cli-build.js`. By example for
-disabling ES6/2015 features you'd do:
+{% highlight js %}
+/* eslint-env node */
+module.exports = {
+  browsers: [
+    'ie 9',
+    'last 1 Chrome versions',
+    'last 1 Firefox versions',
+    'last 1 Safari versions'
+  ]
+};
+{% endhighlight %}
+
+(If these values look familiar, they're the same exact values used by the popular [Autoprefixer](https://github.com/postcss/autoprefixer) project.)
+
+If you need more fine-grained customization over the way that `babel-preset-env` transforms your code,
+simply set any of the options found [here](https://github.com/babel/babel-preset-env#options) on your application's `babel` hash in `ember-cli-build.js`. 
+
+For example, if you wanted to explicitly exclude generator function transpilation from your
+output, your configuration would look like this:
 
 {% highlight js %}
 // ember-cli-build.js
-var EmberApp = require('ember-cli/lib/broccoli/ember-app');
+
+/* eslint-env node */
+'use strict';
+
+const EmberApp = require('ember-cli/lib/broccoli/ember-app');
 
 module.exports = function(defaults) {
-  var app = new EmberApp(defaults, {
```

Comment:
I can see that being a concern, but my thinking was that [keeping the link to the 5.x docs](https://github.com/ember-cli/ember-cli.github.io/pull/141/files#diff-b26e0ee0e3bd8d9f8c34143c96766891R75) would be an adequate way to guide users who haven't upgraded yet. 

Also, this is just my take, but I think one of the main use cases for this documentation, in particular, is to guide someone who's newer to Ember and give them an overall feel for Ember CLI. Thus, they're more likely to be on a newer version -- and we're more likely to help them by linking to relevant projects (in this case, `ember-cli-babel`) for more thorough documentation.